src/scripts/upload-wpt-results.py

```diff
@@ -239,6 +250,7 @@ def notify(url, secret, payload):
 parser.add_argument('--raw-results-directory', required=True)
 parser.add_argument('--platform-id', required=True)
 parser.add_argument('--browser-name', required=True)
+parser.add_argument('--browser-channel', required=True)
```

Comment:
Can you implement `choices: (...)` here as well? If so, then I recommend 👍

Reply:
You bet